### ISSUE TITLE
Document why a React library is needed

### ISSUE THREAD

**Issue by jp7carlos** [FortAwesome/react-fontawesome#223]
I feel like there should be something on the docs that explained why a React library is needed. I guess it's because of code splitting and bundle size, but maybe i'm wrong or maybe other developers will have a hard time figuring it out too. 
The question is: Why not just include the FontAwesome CSS in a more traditional way (className="fa fa-something")?

**Comment by JoeyDye** [comment]
I agree that it should be included in the docs, but until then, this article helps shed some light:

[Using Font Awesome 5 with React](https://scotch.io/tutorials/using-font-awesome-5-with-react)

**Comment by Rycochet** [comment]
It would be nice, but your question itself shows a lack of understanding about how React actually works.

Any React developer will want a package so they can actually use it - the contents of the DOM is supposed to be immutable outside of React, and using JS magic not only breaks that, but also prevents pre-rending static HTML on the server.

**Comment by jp7carlos** [comment]
@Rycochet I was not talking about any JS library, I was talking about using pure CSS. Showing an icon on the screen requires nothing but CSS and the SVG. 

You've gone too far on your real React developer mindset and forgot about the basics.
